### ISSUE TITLE
negative value MIRO loss

### ISSUE THREAD

**Issue by ByungJu-Woo** [kakaobrain/miro#18]
In my setting (fine-tuning pretrained model on our custom dataset), MIRO loss goes to negative value after some iteration.
Is it possible case? or I missed something?
 
pretrained model : CLIP-ViT 
lambda for MIRO loss : 1.0
lr mult for MeanEncoder and VarianceEncoder : 10

**Comment by khanrc** [comment]
Our objective can be negative since it is the lower bound of mutual information.
